src/lib/modules/Document/DocumentEItemUrls.js

```diff
@@ -19,22 +19,29 @@ export class DocumentEItemUrls extends Component {
     );
   };
 
-  userCanSeeFiles = (eitem) => {
+  userCanSeeEItems = (eitem) => {
```

Comment:
I am not sure about this change and below.
From what I remember, it was that when `open_access: False`, links are still visible but not files.
Can you check the issue when this was implemented?

Reply:
You are right. [Here](https://github.com/inveniosoftware/react-invenio-app-ils/issues/223) is the issue. For the `access_urls` of series, when `open_access` is False I am not displaying the urls for not logged in users. Is it right?